websites/B/Brain.fm/presence.ts

```diff
@@ -0,0 +1,108 @@
+const presence = new Presence({
+		clientId: "981442534167441428"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, hostname } = document.location;
+
+	if (["www.brain.fm", "try.brain.fm"].includes(hostname)) {
+		if (["/", "/home"].includes(pathname))
+			presenceData.details = "Viewing the home page";
+		else if (pathname.startsWith("/about"))
+			presenceData.details = "Viewing the about page";
+		else if (pathname.startsWith("/pdfs"))
+			presenceData.details = "Reading a PDF";
+		else if (pathname.startsWith("/science"))
+			presenceData.details = "Viewing the science page";
+		else if (pathname.startsWith("/press"))
+			presenceData.details = "Viewing the press page";
+		else if (pathname.startsWith("/pricing"))
+			presenceData.details = "Viewing pricing options";
+		else if (pathname.startsWith("/jobs"))
+			presenceData.details = "Viewing the jobs page";
+		else if (pathname.startsWith("/faq"))
+			presenceData.details = "Viewing the FAQ page";
+		else if (pathname.startsWith("/contact"))
+			presenceData.details = "Viewing the contact page";
+		else if (pathname.startsWith("/terms"))
+			presenceData.details = "Viewing the terms page";
+		else if (pathname.startsWith("/privacy"))
+			presenceData.details = "Viewing the privacy page";
+		else if (pathname.startsWith("/legal"))
+			presenceData.details = "Viewing the legal page";
+		else if (pathname.startsWith("/forgot-password"))
+			presenceData.details = "Resetting their password";
+	} else if (hostname.startsWith("my")) {
```

Comment:
this can simply be an else, no need for the if
```suggestion
	} else {
```

Reply:
Doing this would cause the code to fail as there is an else statement later on.